<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="9" hostname="Nom-31" failures="4" tests="4" name="_1853Test" time="115.508" errors="0" timestamp="2022-01-30T23:23:38 IST">
  <testcase classname="_1853Test.TC581IndividualEmployeeP60" name="GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott" time="26.545">
    <failure type="java.lang.AssertionError" message="Unable to locate object: By.xpath: //LI[@id=&amp;apos;ctl00_ctl00_ParentContent_SideMenu1_businessMenu&amp;apos;]/A">
      <![CDATA[java.lang.AssertionError: Unable to locate object: By.xpath: //LI[@id='ctl00_ctl00_ParentContent_SideMenu1_businessMenu']/A
at org.testng.Assert.fail(Assert.java:99)
at pages.OpenClient.Click_ClientsClick(OpenClient.java:80)
at _1853Test.TC581IndividualEmployeeP60.GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott(TC581IndividualEmployeeP60.java:90)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott -->
  <testcase classname="_1853Test.TC578NotToSubmit" name="GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott" time="25.877">
    <failure type="java.lang.AssertionError" message="Unable to locate object: By.xpath: //LI[@id=&amp;apos;ctl00_ctl00_ParentContent_SideMenu1_businessMenu&amp;apos;]/A">
      <![CDATA[java.lang.AssertionError: Unable to locate object: By.xpath: //LI[@id='ctl00_ctl00_ParentContent_SideMenu1_businessMenu']/A
at org.testng.Assert.fail(Assert.java:99)
at pages.OpenClient.Click_ClientsClick(OpenClient.java:80)
at _1853Test.TC578NotToSubmit.GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott(TC578NotToSubmit.java:90)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott -->
  <testcase classname="_1853Test.TC580IndividualEmployeePayslip" name="GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott" time="25.239">
    <failure type="java.lang.AssertionError" message="Unable to locate object: By.xpath: //LI[@id=&amp;apos;ctl00_ctl00_ParentContent_SideMenu1_businessMenu&amp;apos;]/A">
      <![CDATA[java.lang.AssertionError: Unable to locate object: By.xpath: //LI[@id='ctl00_ctl00_ParentContent_SideMenu1_businessMenu']/A
at org.testng.Assert.fail(Assert.java:99)
at pages.OpenClient.Click_ClientsClick(OpenClient.java:80)
at _1853Test.TC580IndividualEmployeePayslip.GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott(TC580IndividualEmployeePayslip.java:90)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott -->
  <testcase classname="_1853Test.TC579NotToSubmitEmployeePayslip" name="GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott" time="24.612">
    <failure type="java.lang.AssertionError" message="Unable to locate object: By.xpath: //LI[@id=&amp;apos;ctl00_ctl00_ParentContent_SideMenu1_businessMenu&amp;apos;]/A">
      <![CDATA[java.lang.AssertionError: Unable to locate object: By.xpath: //LI[@id='ctl00_ctl00_ParentContent_SideMenu1_businessMenu']/A
at org.testng.Assert.fail(Assert.java:99)
at pages.OpenClient.Click_ClientsClick(OpenClient.java:80)
at _1853Test.TC579NotToSubmitEmployeePayslip.GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott(TC579NotToSubmitEmployeePayslip.java:90)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlPositiveSelectselectTaxYearsEnterEnterNotesPositiveSelectSubmitReasonClickNott -->
  <testcase classname="tests.TestBase" name="@AfterSuite closeReporter" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite closeReporter -->
  <testcase classname="tests.TestBase" name="@AfterSuite closeReporter" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite closeReporter -->
  <testcase classname="tests.TestBase" name="@AfterSuite closeReporter" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite closeReporter -->
  <testcase classname="tests.TestBase" name="@BeforeSuite setupReporter" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReporter -->
  <testcase classname="tests.TestBase" name="@BeforeSuite setupReporter" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReporter -->
  <testcase classname="tests.TestBase" name="@BeforeSuite setupReporter" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReporter -->
  <testcase classname="tests.TestBase" name="@BeforeSuite allocateData" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite allocateData -->
  <testcase classname="tests.TestBase" name="@BeforeSuite allocateData" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite allocateData -->
  <testcase classname="tests.TestBase" name="@BeforeSuite allocateData" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite allocateData -->
</testsuite> <!-- _1853Test -->
